### DISCUSSION TITLE
Getting NONE in all the fields on the Evaluation server

### DISCUSSION THREAD

**Discussion by animesh-007** [bdd100k/bdd100k#240] [Q&A] [answered]
Hi, I submitted json file of the predictions but I am getting NONE in all the fields. Can anybody let me know how I can resolve this?
<img width="896" alt="Screenshot 2022-05-14 at 5 56 37 PM" src="https://user-images.githubusercontent.com/53366877/168425759-fe79fb59-0174-4922-88bc-35e3f5342b03.png">

**Comment by siyuanliii** [comment, marked as answer]
The None status means your job hasn't been processed. Please wait for a while until it starts.

**Reply by animesh-007** [reply]
Yes, Now it is resolved. I guess there was some server error because of which it took more time than usual. Thanks

**Reply by fabianwindbacher** [reply]
I have the same issue, all Nones, on the Pose Estimation Challenge; however for me an evaluation that was submitted a week ago has still not been processed. (this has worked previously, and then it was done within seconds)

**Reply by CarlHuangNuc** [reply]
> Yes, Now it is resolved. I guess there was some server error because of which it took more time than usual. Thanks

Can you shared how to resolved it?
I encounter the same issue ,but still always None value ?
